fern/pages/sdks/capabilities/auto-pagination.mdx

```diff
@@ -4,7 +4,8 @@ description: Paginate through API responses easily with offset, cursor, and link
 ---
 
 <Warning title='Pro Feature'>
-  This feature is only available on paid plans. [Contact us](https://buildwithfern.com/contact) to get started. 
+  This feature is only available on paid plans. Please schedule a [demo](https://buildwithfern.com/contact) 
+  or [email us](mailto:support@buildwithfern.com) to get started. 
 </Warning>
```

Comment:
would be cool if there was a way of defining this in one place and referencing it on all the pro pages, not but sure if that is possible

Reply:
https://buildwithfern.com/learn/docs/content/reusable-snippets

Reply:
epic, just switched to that